File: dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V3/StachV3Constants.cs
using System.Collections.Generic;

namespace FactSet.Protobuf.Stach.Extensions.V3
{
    internal static class StachV3Constants
    {
        internal const string ColumnSeparator = " | ";
        internal const string GroupPrefix = "group";
        internal const string LevelNameColumn = "level";
        internal const string SecurityNameColumn = "security";
        internal static readonly HashSet<string> IgnoredColumns = new HashSet<string>
        {
            "primary_key",
            "row_path",
            "level",
            "aggregate_rows",
            //"Security Name",
            "pk", 
            "rp", 
            "lvl", 
            "ag"
        };
    }
}

File: java/src/main/java/com/factset/protobuf/stach/extensions/v3/StachV3Constants.java
package com.factset.protobuf.stach.extensions.v3;

import java.util.Arrays;
import java.util.List;

public class StachV3Constants {

	// constants
    public final String columnSeparator = " | ";
    public final String groupPrefix = "group";
    public final String levelNameColumn = "level";
    public final String securityNameColumn = "security";
    public final List<String> ignoredColumns = Arrays.asList("primary_key", "row_path",
            "level",
            "aggregate_rows",
            //"Security Name",
            "pk", "rp", "ag", "lvl");
}
